torch/csrc/distributed/rpc/tensorpipe_agent.cpp

```diff
@@ -0,0 +1,380 @@
+#include <caffe2/torch/csrc/distributed/rpc/tensorpipe_agent.h>
```

Comment:
The path should be relative to `torch`, not `caffe2` for OSS code. This should also help fix the clang-tidy error!